docs/resources/azure_monitor_activity_log_alert.md.erb

```diff
@@ -0,0 +1,83 @@
+---
+title: About the azure_monitor_activity_log_alert Resource
+platform: azure
+---
+
+# azure\_monitor\_activity\_log\_alert
+
+Use the `azure_monitor_activity_log_alert` InSpec audit resource to test properties
+of an Azure Monitor Activity Log Alert.
+
+<br />
+
+## Syntax
+
+An `azure_monitor_activity_log_alert` resource block identifies a Activity Log Alert by
+name and resource group.
+
+    describe azure_monitor_activity_log_alert(resource_group: 'example', name: 'alert-name') do
```

Comment:
Can we switch the argument format to mix case? In the other docs I would have done `AlertName` instead of `alert-name`.